content/docs/2.9/operate/cluster.md

```diff
@@ -130,3 +130,29 @@ To specify values other than their defaults, you can set the following environme
 | KEDA_METRICS_LEADER_ELECTION_LEASE_DURATION  | Metrics Server | LeaseDuration    |
 | KEDA_METRICS_LEADER_ELECTION_RENEW_DEADLINE  | Metrics Server | RenewDeadline    |
 | KEDA_METRICS_LEADER_ELECTION_RETRY_PERIOD    | Metrics Server | RetryPeriod      |
+
+## Restrict Secret Access
+
+By default, KEDA requires adding `secrets` to clusterrole as following:
+```yaml
+- apiGroups:
+  - ""
+  resources:
+  - external
+  - pods
+  - secrets
+  - services
+  verbs:
+  - get
+  - list
+  - watch
+```
+However, it might lead to security risk especially in production environment, since it will grant permission to read `secrets` from all namespaces.
```

Comment:
```suggestion
However, this might lead to security risk (especially in production environment) since it will grant permission to read `secrets` from all namespaces.
```